### PR TITLE
Stock API

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR is about the stock API implementation : 

- [x] Database Design
- [x] Server side API
   - [x] Create a new stock lots
   - [x] Create a stock movements
   - [x] Get list of data
   - [x] Provide API for stock documents
- [x] Integration Test
   - [x] Create a new stock lots
   - [x] Create a stock movements

### CODE REVIEW

File: test/integration/stock.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      .catch(helpers.handler);
+  });
+
+  // create stock movement to depot 
+  it('POST /stock/lots/movements distribute stock lots to a depot', () => {
+    return agent.post('/stock/lots/movements')
+      .send(movement_depot)
+      .then((res) => {
+        helpers.api.created(res);
+        console.log(res.body.uuid)
```
Comment (mbayopanda) [inline test/integration/stock.js]: `console.log(res.body.uuid)` is not necessary and it'll be removed

**Comment by jniles** [comment]
@mbayopanda, what should we do with this PR since #1353 exists?

File: client/src/less/bhima-bootstrap.less [hunk 2]
```diff
@@ -263,3 +263,9 @@ input[type=number] {-moz-appearance: textfield;}
   animation-name: error-flash;
   animation-duration: 2s;
 }
+
+/* large button as menu */
+.bg-ima-dark-blue { background: @ima-dark-blue; }
```
Comment (jniles) [inline client/src/less/bhima-bootstrap.less:268]: :+1:

**Comment by mbayopanda** [comment]
Code in the `feature-stock-management` branch

**Pull request closed without merging**